reporting/api/src/main/java/org/jboss/windup/reporting/model/ClassificationModel.java

```diff
@@ -97,4 +104,12 @@
     @SetInProperties(propertyPrefix = "tag")
     Set<String> getTags();
 
+    abstract class Impl implements ClassificationModel, JavaHandlerContext<Vertex>
```

Comment:
Ok I wasn't sure how java handles overriding method definition and somehow expected such a way to cause another problems. I will remake it then, thanks!